src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -0,0 +1,96 @@
+(ns status-im2.contexts.onboarding.common.carousel.view
+  (:require [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [react-native.navigation :as navigation]
+            [react-native.reanimated :as reanimated]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.onboarding.common.carousel.style :as style]
+            [status-im2.contexts.onboarding.common.carousel.animation :as animation]))
+
+(def header-text
+  [{:text     (i18n/label :t/join-decentralised-communities)
+    :sub-text (i18n/label :t/participate-in-the-metaverse)}
+   {:text     (i18n/label :t/chat-with-friends)
+    :sub-text (i18n/label :t/with-full-encryption)}
+   {:text     (i18n/label :t/own-your-crypto)
+    :sub-text (i18n/label :t/use-the-multichain-wallet)}
+   {:text     (i18n/label :t/discover-web3)
+    :sub-text (i18n/label :t/explore-the-decentralized-web)}])
+
+(defn header-text-view
+  [index window-width]
+  [rn/view {:style (style/header-text-view window-width)}
+   [quo/text
+    {:style  style/carousel-text
+     :weight :semi-bold
+     :size   :heading-2}
+    (get-in header-text [index :text])]
+   [quo/text
+    {:style style/carousel-sub-text
+     :size  :paragraph-1}
+    (get-in header-text [index :sub-text])]])
+
+(defn content-view
+  [window-width status-bar-height index]
+  (let [content-width (* 4 window-width)]
+    [:<>
+     [rn/image
+      {:style  (style/background-image (* 4 window-width))
```

Comment:
`(style/background-image content-width)` instead of `(style/background-image (* 4 window-width))`?